libs/coin-polkadot/src/types/model.fixture.ts

```diff
@@ -0,0 +1,109 @@
+import BigNumber from "bignumber.js";
+import { faker } from "@faker-js/faker";
+import { listCryptoCurrencies } from "@ledgerhq/cryptoassets/currencies";
+import { PolkadotAccount, PolkadotOperationExtra, PolkadotResources, Transaction } from "../types";
+import { getAbandonSeedAddress } from "../../../ledgerjs/packages/cryptoassets/lib";
+import { Operation } from "@ledgerhq/types-live";
+
+export function createFixtureAccount(account?: Partial<PolkadotAccount>): PolkadotAccount {
```

Comment:
I'm more of the "specific mock for specific method team", using global fixtures can end up with you editing the fixture output in your test anyway, as you are using "more than what's needed" and need to target specific attributes usually.

If you use global fixture it can also hide uncovered execution branches that you should be aware of

Example : 
- Imagine a method checking if creationDate of your account is recent we can either use a global createMockAccount method and edit its creationDate attribute or provide a { creationDate: value } mock version as input
- If there is another attribute usage that you aren't aware of (because the method is complex) you won't notice it as you are using a global fixture (example is bad in that case as it's pretty straightforward)